Makefile

```diff
@@ -178,12 +179,13 @@ else
   ONLOAD_CFLAGS += -DCI_HAVE_SFC=0
 endif
 
+KO_DIRS := $(patsubst %,$(KBUILDTOP)/%/*.ko,$(DRIVER_SUBDIRS))
 
 .PHONY: modules modules_install clean_kernel kernel
 
 kernel: modules
 	@mkdir -p $(KBUILDTOP)/driver/linux
-	$(Q)ln -rsf $(KBUILDTOP)/src/driver/linux_onload/*.ko $(KBUILDTOP)/src/driver/linux_char/*.ko $(KBUILDTOP)/src/driver/linux_resource/*.ko $(KBUILDTOP)/src/driver/linux_net/drivers/net/ethernet/sfc/*.ko $(KBUILDTOP)/driver/linux
+	$(Q)ln -rsf $(KO_DIRS) $(KBUILDTOP)/driver/linux
```

Comment:
I reckon just wrapping the original file list in `$(wildcard)` would have worked too, but this way is more explicit